src/utils/nodeTypeProcessing.ts

```diff
@@ -348,20 +362,23 @@ export function getPackedByteSize(type: TypeNode, version: string): number | big
   }
 
   if (type instanceof ArrayType && type.size !== undefined) {
-    return type.size * BigInt(getPackedByteSize(type.elementT, version));
+    return type.size * BigInt(getPackedByteSize(type.elementT, inference));
   }
 
   const sumMemberSize = (acc: bigint, cv: TypeNode): bigint => {
-    return acc + BigInt(getPackedByteSize(cv, version));
+    return acc + BigInt(getPackedByteSize(cv, inference));
   };
   if (type instanceof TupleType) {
     return type.elements.reduce(sumMemberSize, 0n);
   }
 
   if (type instanceof UserDefinedType && type.definition instanceof StructDefinition) {
-    assert(version !== undefined, 'Struct byte size calculation requires compiler version');
+    assert(
+      inference.version !== undefined,
+      'Struct byte size calculation requires compiler version',
+    );
```

Comment:
The old assert instruction was a leftover, is safe to remove it (no need to update it)